### ISSUE TITLE
[Supporting the platform – Make Code – Home page ] ‘Micro bit & Microsoft’ logo are getting disappear after applying high contrast white theme  under ‘Home Page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3441]
**User Experience:**
Visually impaired user will be impacted severely if ‘Micro bit & Microsoft’ logo does not adapt high contrast(White) theme as they won't be able to take action on that controls and he/she would miss the functionality of such control or element.

**Note**: User credentials should NOT be included in the bug.

**Pre-requisites:**
Enable high contrast mode. Settings->Ease of Access->Color & High contrast

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/#
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify that ' Micro bit & Microsoft’ logo controls are getting disappear after applying high contrast white theme or not.

**Actual Result:**
‘Micro bit & Microsoft’ logo are getting disappear after applying high contrast white theme  under ‘Home Page’, due to this user is unable to see the control.

**Expected Result:**
‘Micro bit & Microsoft’ logo controls should be visible after applying high contrast white theme under home page, so that user could easily see the controls.

**MAS Reference:**
[MAS 4.3.1 - No Disruption of Accessibility Features](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74})

**Reference Links:**
•	[Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/PageNotFoundError.aspx?requestUrl=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
•	External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [Link](https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx). “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
•	Please reach out to [C&AI Teams channel](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2F_%23%2Fl%2Fteam%2F19%3A3cd071821b694f11b054d878453cd75f%40thread.skype%2Fconversations%3FgroupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&type=team&deeplinkId=99c73412-3928-459b-a051-ad90788e6969&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true) for any process related queries.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
#URL:https://makecode.microbit.org/#

**Comment by abchatra** [comment]
@Shalini3457 please add screenshots. I do see these logos

**Comment by abchatra** [comment]
Also these issues don't have information about the browser.
